src/main/java/edu/kit/provideq/toolbox/Bound.java

```diff
@@ -0,0 +1,11 @@
+package edu.kit.provideq.toolbox;
+
+/**
+ * Represents a bound estimation for the solution of a problem.
+ *
+ * @param bound the bound value
+ * @param boundType the type of the bound
+ * @param executionTime the time it took to estimate the bound
+ */
+public record Bound(String bound, BoundType boundType, long executionTime) {
```

Comment:
This could use an alternative constructor which sets the execution time to -1 by default as estimator implementations don't set the execution time, just the Problem class.